gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -46,11 +82,50 @@
 
     public static final String REFASTER_CONFIGURATION = "refaster";
     public static final String EXTENSION_NAME = "baselineErrorProne";
+    public static final String PROP_STRICT = "com.palantir.baseline-error-prone.strict";
 
     private static final String ERROR_PRONE_JAVAC_VERSION = "9+181-r4173-1";
     private static final String PROP_ERROR_PRONE_APPLY = "errorProneApply";
     private static final String PROP_REFASTER_APPLY = "refasterApply";
 
+    private static final ImmutableSet<BugCheckerInfo> allBaselineChecks =
+            BuiltInCheckerSuppliers.getSuppliers(ImmutableList.of(
+            CatchBlockLogException.class,
+            DangerousCompletableFutureUsage.class,
+            DangerousJsonTypeInfoUsage.class,
+            DangerousParallelStreamUsage.class,
+            DangerousStringInternUsage.class,
+            DangerousThreadPoolExecutorUsage.class,
+            DangerousThrowableMessageSafeArg.class,
+            GradleCacheableTaskAction.class,
+            GuavaPreconditionsMessageFormat.class,
+            JUnit5RuleUsage.class,
+            LambdaMethodReference.class,
+            LogSafePreconditionsMessageFormat.class,
+            NonComparableStreamSort.class,
+            OptionalOrElseMethodInvocation.class,
+            OptionalOrElseThrowThrows.class,
+            PreconditionsConstantMessage.class,
+            PreferBuiltInConcurrentKeySet.class,
+            PreferCollectionTransform.class,
+            PreferListsPartition.class,
+            PreferSafeLoggableExceptions.class,
+            PreferSafeLoggingPreconditions.class,
+            PreventTokenLogging.class,
+            ShutdownHook.class,
+            Slf4jConstantLogMessage.class,
+            Slf4jLogsafeArgs.class,
+            SwitchStatementDefaultCase.class,
+            ValidateConstantMessage.class
+    ));
```

Comment:
Could we use reflection to build this list? I'm concerned that we'll add a new class and forget to add it here.

Reply:
Updated to use `ServiceLoader.load(BugChecker.class, BaselineErrorProne.class.getClassLoader())` filtered down to `com.palantir.**` classes. Thoughts on this approach?